test/NeoSharp.Core.Test/Network/UtServer.cs

```diff
@@ -97,23 +86,27 @@ public void Start_PeerConnectionThrowException_WarningMessageIsLoggedServerKeepL
         public void Stop_SuccessfulPeerConnection_StoppingServerLeadsToDisconnectingPeer()
         {
             // Arrange 
-            var peerMock = AutoMockContainer.GetMock<IPeer>();
+            var endpoint = new EndPoint { Host = "localhost", Protocol = Protocol.Tcp, Port = 8081 };
```

Comment:
Imho, something wrong with API if we define method for local endpoint but are forced to create this still manuallly

Reply:
the tests still need work to be readable ... this is not yet the final version.
right now there still a changes in the server class and make sense to have all the setup in the test. As soon the code in the testee is stable we can start with optimize the test code.

Reply:
I was under impression:
1. My refactoring provided a shareable initialization logic and helped to reduce duplicated lines.
2. IBlockchain was required to be configured and without this tests would fail.

Reply:
There is no reason to have that impression ... optimizations will be done as soon we have the Server base code stable and then we know where we can optimize.